OrbitCore/SamplingProfiler.h

```diff
@@ -19,73 +19,73 @@ class Process;
 struct SampledFunction {
   SampledFunction() = default;
 
-  std::string m_Name;
-  std::string m_Module;
-  std::string m_File;
-  float m_Exclusive = 0;
-  float m_Inclusive = 0;
-  int m_Line = 0;
-  uint64_t m_Address = 0;
-  orbit_client_protos::FunctionInfo* m_Function = nullptr;
+  std::string name;
+  std::string module;
+  std::string file;
+  float exclusive = 0;
+  float inclusive = 0;
+  int line = 0;
+  uint64_t address = 0;
+  orbit_client_protos::FunctionInfo* function = nullptr;
 };
 
 struct ThreadSampleData {
-  ThreadSampleData() { m_ThreadUsage.push_back(0); }
+  ThreadSampleData() { thread_usage.push_back(0); }
   [[nodiscard]] uint32_t GetCountForAddress(uint64_t address) const;
-  std::unordered_map<CallstackID, uint32_t> m_CallstackCount;
-  std::unordered_map<uint64_t, uint32_t> m_AddressCount;
-  std::unordered_map<uint64_t, uint32_t> m_RawAddressCount;
-  std::unordered_map<uint64_t, uint32_t> m_ExclusiveCount;
-  std::multimap<uint32_t, uint64_t> m_AddressCountSorted;
-  uint32_t m_NumSamples = 0;
-  std::vector<SampledFunction> m_SampleReport;
-  std::vector<float> m_ThreadUsage;
-  float m_AverageThreadUsage = 0;
-  ThreadID m_TID = 0;
+  std::unordered_map<CallstackID, uint32_t> callstack_count;
+  std::unordered_map<uint64_t, uint32_t> address_count;
+  std::unordered_map<uint64_t, uint32_t> raw_address_count;
+  std::unordered_map<uint64_t, uint32_t> exclusive_count;
```

Comment:
maybe replace these with flag_hash_maps?

Reply:
agree maybe this is a good point in time to do this.

Reply:
Updated the PR: replaced std::unordered_map/std::unordered_set with absl::flat_hash_map/absl::flat_hash_set